packages/astroport-ibc/src/lib.rs

```diff
@@ -1,3 +1,6 @@
 use std::ops::RangeInclusive;
 
 pub const TIMEOUT_LIMITS: RangeInclusive<u64> = 60..=43200;
+
+/// 2 weeks - 2 months
+pub const SIGNAL_OUTAGE_LIMITS: RangeInclusive<u64> = 1209600..=5184000;
```

Comment:
I think 2 weeks - 2 months is a reasonable timeframe. I assume we'll be running a bot to send a heartbeat once a week. 

If we assume we use 2 weeks and were to send the heartbeat and realise that the client has expired, we'll have about 1 week to correct it. Most chains have a 1 week governance proposal voting period, however, Neutron has a 2 week voting period. 

Could that create a situation where ownership could be changed due to a timing mismatch for the proposals? Or is the idea that if that were to happen, ownership should be changed and updated again once the clients are back up?

Reply:
Yes, that could and we should choose an appropriate limit for an outpost, e. g. 2+ weeks for Neutron, initially one month was considered for all outposts. If ownership is changed, we expect that the emergency owner will do everything needed to recover the normal governance flow.